akka-persistence-query/src/main/scala/akka/persistence/query/journal/inmem/internal/EventsByPersistenceIdStage.scala

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.query.journal.inmem.internal
+
+import akka.actor.ActorRef
+import akka.annotation.InternalApi
+import akka.persistence.JournalProtocol.RecoverySuccess
+import akka.persistence.JournalProtocol.ReplayMessagesFailure
+import akka.persistence.Persistence
+import akka.persistence.journal.inmem.InmemJournal
```

Comment:
I expected that this would be on top of the new inmem journal in `akka-persistence-testkit` so that it can be used with `PersistenceTestKit` and `EventSourcedBehaviorTestKit`. It could be placed in `akka-persistence-testkit`.